sdk/face/ai-vision-face-rest/README.md

```diff
@@ -0,0 +1,431 @@
+
+```bash
+# Get the API keys for the Face resource
+az cognitiveservices account keys list --name "<resource-name>" --resource-group "<resource-group-name>"
+```
+
+#### Create the client with AzureKeyCredential
+
+To use an API key as the `credential` parameter, pass the key as a string into an instance of [AzureKeyCredential](https://learn.microsoft.com/javascript/api/@azure/core-auth/azurekeycredential?view=azure-node-latest).
+
+```js
+import { AzureKeyCredential } from '@azure/core-auth';
+import createFaceClient from '@azure-rest/ai-vision-face';
+
+const endpoint = process.env['FACE_ENDPOINT'] || '<endpoint>';
+const apikey = process.env['FACE_APIKEY'] || '<apikey>';
+const credential = new AzureKeyCredential(apikey);
+const client = createFaceClient(endpoint, credential);
+```
+
+#### Create the client with an Azure Active Directory credential
+
+`AzureKeyCredential` authentication is used in the examples in this getting started guide, but you can also authenticate with Azure Active Directory using the [@azure/identity](https://learn.microsoft.com/javascript/api/@azure/identity/?view=azure-node-latest) library.
+Note that regional endpoints do not support AAD authentication. Create a [custom subdomain](https://docs.microsoft.com/azure/cognitive-services/authentication#create-a-resource-with-a-custom-subdomain) name for your resource in order to use this type of authentication.
+
+To use the [DefaultAzureCredential](https://learn.microsoft.com/javascript/api/@azure/identity/defaultazurecredential?view=azure-node-latest) type shown below, or other credential types provided with the Azure SDK, please install the `@azure/identity` package:
+
+```
+npm install --save @azure/identity
+```
+
+You will also need to [register a new AAD application and grant access](https://docs.microsoft.com/azure/cognitive-services/authentication#assign-a-role-to-a-service-principal) to Face by assigning the `"Cognitive Services User"` role to your service principal.
+
+Once completed, set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables:
+`AZURE_CLIENT_ID`, `AZURE_TENANT_ID`, `AZURE_CLIENT_SECRET`.
+
+```js
+/**
+ * DefaultAzureCredential will use the values from these environment
+ * variables: AZURE_CLIENT_ID, AZURE_TENANT_ID, AZURE_CLIENT_SECRET
+ */
+import { DefaultAzureCredential } from '@azure/identity';
+import createFaceClient from '@azure-rest/ai-vision-face';
+
+const endpoint = process.env['FACE_ENDPOINT'] || '<endpoint>';
+const credential = new DefaultAzureCredential();
+const client = createFaceClient(endpoint, credential);
+```
+
+## Key concepts
+
+### FaceClient
+
+The `FaceClient` is the primary interface for developers interacting with the Azure AI Face service. It serves as the gateway from which all interaction with the library will occur.
+
+### Long-running operations
```

Comment:
I am curious why we need this part? I see there's example in the below that shows our customer how to use long-running operations already, and FYI, there's a section about long running operation in our RLC general blog https://devblogs.microsoft.com/azure-sdk/azure-rest-libraries-for-javascript

Reply:
Agreed. I prefer guiding the customer to learn LRO concept and helper to our guide.

But in our README I prefer to keep the part of introducing what methods are LRO in FACE client.

![image](https://github.com/Azure/azure-sdk-for-js/assets/9943211/dbd32ed2-0f07-4c3e-a539-5d1c158196f5)